### DISCUSSION TITLE
Integration in React for each route[Question]

### DISCUSSION THREAD

**Discussion by Hamid293** [umami-software/umami#1197] [Q&A]
Hey, thank you for creating and open-sourcing this.

I am thinking of collecting analytics for my website which is made in React. Can someone guide me on how to add umami tracker in React? 
Also, I want to create a separate dashboard for each route on my site. What should be the ideal setup for each route to have its own dashboard showing its analytics?

Thank you

**Comment by mikecao** [comment]
You would add the tracker code to your head tag like you do any other website.

If you want to track routes separately, just create a tracker for each then embed it into the route page.
